modyn/supervisor/internal/eval/handler.py

```diff
@@ -1,15 +1,74 @@
 from __future__ import annotations
 
-from dataclasses import dataclass
-
+import pandas as pd
 from modyn.config.schema.pipeline import EvalHandlerConfig
+from modyn.supervisor.internal.eval.strategies.abstract import AbstractEvalStrategy
+from modyn.utils import dynamic_module_import
+from pydantic import BaseModel
+
+eval_strategy_module = dynamic_module_import("modyn.supervisor.internal.eval.strategies")
+
+
+class EvalRequest(BaseModel):
```

Comment:
Any reason for using pydantic for internal state here as well instead of the previously used dataclass?

Reply:
I originally used a dataclass but then realized that I want to log exactly the content of this structure (previously this was all duplicated in a pydantic logging model). I decided to just make this a pydantic model and use it for logging directly.